backend/src/main/java/com/redhat/cloud/notifications/routers/OrgConfigResource.java

```diff
@@ -0,0 +1,63 @@
+package com.redhat.cloud.notifications.routers;
+
+import com.redhat.cloud.notifications.Constants;
+import com.redhat.cloud.notifications.auth.ConsoleIdentityProvider;
+import com.redhat.cloud.notifications.db.repositories.AggregationOrgConfigRepository;
+import com.redhat.cloud.notifications.models.AggregationOrgConfig;
+import io.quarkus.logging.Log;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.annotation.security.RolesAllowed;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+import java.time.LocalTime;
+
+import static com.redhat.cloud.notifications.routers.SecurityContextUtil.getOrgId;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Path(Constants.API_NOTIFICATIONS_V_1_0 + "/org-config")
+public class OrgConfigResource {
```

Comment:
@josejulio This class contains new REST endpoints that will be used to store/retrieve the aggregation execution time for each org. I'm not sure if we should include these endpoints in the OpenAPI spec or hide them. WDYT?

Reply:
I'm merging before your answer. We'll add the annotations if needed afterwards.